server/lorax_server/models/custom_modeling/flash_bert_modeling.py

```diff
@@ -164,7 +164,7 @@ def forward(self, hidden_states, cu_seqlens, max_s):
         q, k, v = qkv.view(-1, self.num_heads * 3, self.head_size).split(self.num_heads, dim=1)
 
         attn_output = torch.empty_like(q)
-        attention(q, k, v, attn_output, cu_seqlens, max_s, self.softmax_scale)
+        attention(q, k, v, None, None, cu_seqlens, max_s, self.softmax_scale)
```

Comment:
Did you verify this works? You're not modifying the `attn_output`, which is now returned from the attention function, so I suspect this will effectively be a no-op.

Reply:
ah man good catch!

Reply:
fixed - good catch